prow/plugins/approve/approve.go

```diff
@@ -139,12 +140,13 @@ func handleGenericCommentEvent(pc plugins.Agent, ce github.GenericCommentEvent)
 		pc.Logger,
 		pc.GitHubClient,
 		pc.OwnersClient,
+		pc.Config,
 		pc.PluginConfig,
 		&ce,
 	)
 }
 
-func handleGenericComment(log *logrus.Entry, ghc githubClient, oc ownersClient, config *plugins.Configuration, ce *github.GenericCommentEvent) error {
+func handleGenericComment(log *logrus.Entry, ghc githubClient, oc ownersClient, prowConfig *config.Config, config *plugins.Configuration, ce *github.GenericCommentEvent) error {
```

Comment:
Please just pass the substructure that is used by the handlers (`config.GitHubOptions`) instead of the entire config.

Reply:
Makes sense... done.